winsup/utils/getprocaddr.c

```diff
@@ -27,13 +27,14 @@ static HANDLE CtrlEvent;
 static int
 inject_remote_thread_into_process (HANDLE process,
                                    LPTHREAD_START_ROUTINE address,
-                                   uintptr_t exit_code)
+                                   uintptr_t exit_code,
+                                   DWORD * thread_return)
```

Comment:
Isn't Cygwin's code style cuddling the asterisk with the variable name?

Reply:
Damned if I know.  I'll change it though